browser/src/control/Control.UIManager.js

```diff
@@ -1570,6 +1570,9 @@ L.Control.UIManager = L.Control.extend({
 				that.closeModal(dialogId);
 			}}
 		]);
+
+		// focus on input modal on show
+		document.getElementById('input-modal-input').focus();
```

Comment:
We should try to avoid `getElementById` where possible.
Also: why not put this inside `showModal` function? then in all places where we show modal we will focus it, not only in this case

Reply:
if we have to, better to query inside container: `container.querySelector('#id')`
that makes us sure we will not get element from some other place than our current component/dialog

Reply:
@eszkadev found better way :)

It seems it was a regression from my commit when i did fix for tabcontrol focus https://github.com/CollaboraOnline/online/pull/10795